README.md

```diff
@@ -1,2 +1,36 @@
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Technical-design-Page
-Technical design Page for Web Development Technologies
+# Technical design Page for Web Development Technologies
```

Comment:
Add title to readme file

Reply:
- [ ]  As the previous reviewer noted, you have to explain why these changes are necessary and make the request clear 😁 . In this suggestion, It is not quite clear why it is useful 🤔.
- [ ] [OPTIONAL] It would  also be good to use emojis, they can make your requests more friendly 🤗

Reply:
- Review: I suggest you add a title to the readme file.
- Reason: As part of Microverse best practice, it is good that you have a Readme a file which provides information about about the project you have done. A project title will give the use an idea of what your code is all about.